{
    "checksum": "e7032cccd47013dcb044021c6dfbb09ef496a02120a8e023ffebb7c2c6bbfae9",
    "file": "error.c",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-c"
    },
    "number_of_duplicates": 3,
    "package": "pytsk",
    "path": "pytsk/20200117-3/error.c",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/pytsk/20200117-3/",
        "metric": {
            "size": 2356
        },
        "pts_link": "https://tracker.debian.org/pkg/pytsk",
        "sloc": [
            [
                "ansic",
                3090
            ],
            [
                "python",
                3007
            ],
            [
                "sh",
                115
            ],
            [
                "makefile",
                68
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/pkg-security-team/pytsk",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/pytsk/20200117-3/error.c",
    "stat": {
        "perms": "rw-r--r--",
        "size": 3077,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "20200117-3"
}